expyfun/io/tests/test_parse.py

```diff
@@ -45,3 +47,59 @@ def test_parse():
     data = read_tab(ec.data_fname, group_end=None)
     assert_equal(len(data[0]['misc']), 2)  # includes between-trials stuff
     assert_equal(len(data[1]['misc']), 2)
+
+
+def test_reconstruct():
+    """Test Tracker objects reconstruction"""
+
+    # test with one TrackerUD
+    with ExperimentController(*std_args, stim_fs=44100, **std_kwargs) as ec:
```

Comment:
I copied and pasted from `test_parse()`, but upon further (initial) thought, I don't see why it should be.